index.js

```diff
@@ -0,0 +1,106 @@
+// First, make sure to make all buttons for caclautor,
+// meaning numbers, getTotal(),getHistory(), clear, undo,operants
+// replace cuurent js code with previous js code
+// Make a current operant and previous operant attribute  on screen, both equal to ""
+// for each number click, concatnate the value string of that number to the current operant
+// maybe use .innerHtml or something reaserch what you can use
+// then when an operant is clicked, cuurent operasant is now previous operant and
+// then modfiy current operant to be "". also save the operant that was clicked on a varaible
+// then when numbers are clicked, it should be concatnated to currentOperant.
+// then when equalSButton  is clicked, we done a switch staement to check which operant is used and
+// use corresponding function on calcualtor to calculate previousopprant, currentOperant with the operant
+// then call .getTotal()
+
+const screen = document.querySelector("#screen");
+const numbers = document.querySelectorAll("[numbersToPress]");
+const operationButtons = document.querySelectorAll(".operationButtons");
```

Comment:
why are the operation buttons selected using classes and the numbersToPress based on attributes?

Also we typically use kebab-case, not camelCase

Reply:
Good point, It would be so much cleaner If they were both classes. 
Alright, changed them to kebab-case.